CHANGELOG/unreleased/kong/11484.yaml

```diff
@@ -0,0 +1,7 @@
+message: "Tracing: fix an issue that resulted in some parent spans to end before their children due to lack of precision"
```

Comment:
Could you explain in what condition the low precision leads to a child's span with an earlier ending time?

Reply:
An example is visible in the screenshots of https://github.com/Kong/kong/issues/11294, in general, since the balancer's span end time is recorded with nanosecond precision (e.g. 1,693,916,282,000,123,456) and the request (root / kong) span's end time **was** recorded with millisecond precision (e.g. 1,693,916,282,000) we could expect to see the request's end up to 999,999ns (0.999999 ms) earlier than the balancer.

Reply:
Got it. So maybe we should name the problem like "mixing of different precisions"?

Reply:
@StarlightIbuki I rephrased the changelog message to make it more clear, thanks